### DISCUSSION TITLE
bind-address and traefik 80 port iptables

### DISCUSSION THREAD

**Discussion by gytisgreitai** [k3s-io/k3s#11178] [Q&A]
Hey,

I have a debian host where I want to run server node.
That host  besides other non containerised workloads currently runs nginx on port 80 and until I add more agents and migrate all workloads to them I want to keep it that way
I added an IP alias on that host (192.168.3.170) and made nginx only bind to 192.168.3.4 80 port.

I used k3s install and added bind-address and other ip params:
```
curl -sfL https://get.k3s.io | INSTALL_K3S_EXEC="--bind-address=192.168.3.170 --advertise-address=192.168.3.170 --node-ip=192.168.3.170 --node-external-ip=192.168.3.170" sh -s -
```

But `iptables -t nat` does not seem to care about my wish to bind everything to `192.168.3.170` and leave `192.168.3.4` alone.

I could run nginx on that server but I would like for the time being to keep my nginx on the host instead of running on kubernetes server node

Any hints on how to solve this issue (if it's possible at all)?

**Comment by brandond** [comment]
Bind address just sets the address that the embedded apiserver and kubelet bind to, which run on the host itself. It does not in any way impact things that you run within the cluster.
